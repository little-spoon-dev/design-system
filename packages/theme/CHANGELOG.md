# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [1.13.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.13.0...@littlespoon/theme@1.13.1) (2022-08-11)

### Bug Fixes

- replace variant with color props ([3f2f0cf](https://github.com/little-spoon-dev/design-system/commit/3f2f0cf86144616e853b8410cba2aef15e9fd81c))
- spread colors in export ([92681e1](https://github.com/little-spoon-dev/design-system/commit/92681e112bda4ba7efae51240b646307019ef3b7))
- use span element for label ([f2d87dc](https://github.com/little-spoon-dev/design-system/commit/f2d87dc44f21b3aa1445331b057ea7b9da1934b7))

## [1.13.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.12.0...@littlespoon/theme@1.13.0) (2022-07-15)

### Features

- **theme:** add style prop sx ([e975478](https://github.com/little-spoon-dev/design-system/commit/e975478b5c49ab42f78a237cc0ecd4b686cb9d91))

## [1.12.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.11.1...@littlespoon/theme@1.12.0) (2021-11-13)

### Features

- **theme:** duplicate and rename primary color brand blue LS-2313 ([31d48b9](https://github.com/little-spoon-dev/design-system/commit/31d48b9aa8f1988d4be011fbe074a0eeb30eba54))

## [1.11.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.11.0...@littlespoon/theme@1.11.1) (2021-11-13)

**Note:** Version bump only for package @littlespoon/theme

## [1.11.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.10.0...@littlespoon/theme@1.11.0) (2021-10-28)

### Features

- **theme:** make breakpoints up and down last arg optional LS-2111 ([caa388e](https://github.com/little-spoon-dev/design-system/commit/caa388e75e319c95207e1c1b21b3174f3dcb2428))

## [1.10.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.9.0...@littlespoon/theme@1.10.0) (2021-10-27)

### Features

- **theme:** update breakpoint xs to 375 and xxl to 1440 LS-2102 ([321dc66](https://github.com/little-spoon-dev/design-system/commit/321dc66305b467e7607f24a8bfa491309763bdba))

## [1.9.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.8.0...@littlespoon/theme@1.9.0) (2021-10-22)

### Features

- **theme:** add breakpoints LS-2016 ([5b02162](https://github.com/little-spoon-dev/design-system/commit/5b021628d6258ffe983bd85079465916e3b9516f))
- **theme:** add primary font alias caption1 LS-2016 ([c249d1d](https://github.com/little-spoon-dev/design-system/commit/c249d1d4d1776f5d3479ccccaf6c4b01fd99c274))
- **theme:** rename display large & medium to display1 & display2 LS-2016 ([5ce7147](https://github.com/little-spoon-dev/design-system/commit/5ce7147abab040a78225339c49453ebe05e482e1))

## [1.8.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.7.0...@littlespoon/theme@1.8.0) (2021-10-13)

### Features

- **theme:** export p1, p2, p3, p4 from fonts paragraph LS-2014 ([58279a9](https://github.com/little-spoon-dev/design-system/commit/58279a9ef752f39dea9120a7101b6f7ee5b45365))

## [1.7.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.6.0...@littlespoon/theme@1.7.0) (2021-10-13)

### Features

- **theme:** modularize theme fonts primary and secondary LS-2000 ([8591915](https://github.com/little-spoon-dev/design-system/commit/8591915f51584b0fd4870f247b51a6abc4425c96))

## [1.6.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.5.0...@littlespoon/theme@1.6.0) (2021-10-13)

### Features

- **breadcrumbs:** add breadcrumbs to design system, add visited link color LS-1919 ([6a5a2c1](https://github.com/little-spoon-dev/design-system/commit/6a5a2c1cf6849b7897f465d8cb75eb3b29a789a0))

## [1.5.0](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.4.1...@littlespoon/theme@1.5.0) (2021-10-04)

### Features

- **theme:** add sub surface dull color LS-1890 ([dd65e72](https://github.com/little-spoon-dev/design-system/commit/dd65e72dec0d91bfb2b1d054dcc7ae55da447324))

## [1.4.1](https://github.com/little-spoon-dev/design-system/compare/@littlespoon/theme@1.4.0...@littlespoon/theme@1.4.1) (2021-10-04)

**Note:** Version bump only for package @littlespoon/theme

## [1.4.0](https://www.github.com/little-spoon-dev/design-system/compare/theme-v1.3.1...%40littlespoon%2Ftheme%401.4.0) (2021-09-23)

### Features

- **theme:** add additional font sizes LS-1757 ([2229b83](https://github.com/little-spoon-dev/design-system/commit/2229b838601d0bcd4c6dc3173690460cb37e9f42))
- **theme:** update font variants to be objects ([e775d3f](https://github.com/little-spoon-dev/design-system/commit/e775d3f33dd4c30258149f6e905f1f5ff177233f))
- **theme:** update stories with font variants ([af382d7](https://github.com/little-spoon-dev/design-system/commit/af382d726928565a2274f90e5e210d14e63f9037))

## [1.3.1](https://www.github.com/little-spoon-dev/design-system/compare/theme-v1.3.0...theme-v1.3.1) (2021-09-23)

### Performance Improvements

- **theme:** memoize rgb util LS-1802 ([048f397](https://www.github.com/little-spoon-dev/design-system/commit/048f3975f7692038b284fb1b6790804ba2e2a0a9))

## [1.3.0](https://www.github.com/little-spoon-dev/design-system/compare/theme-v1.2.0...theme-v1.3.0) (2021-09-20)

### Features

- **theme:** update primary and secondary colors LS-1776 ([60681c5](https://www.github.com/little-spoon-dev/design-system/commit/60681c564e533bf6852eb995d30b04b4cc7b0d4c))

## [1.2.0](https://www.github.com/little-spoon-dev/design-system/compare/theme-v1.1.0...theme-v1.2.0) (2021-09-17)

### Features

- **theme:** add shade tokens to theme ([ce16bac](https://www.github.com/little-spoon-dev/design-system/commit/ce16bacb53d5653e99abcd5ebea14ca7503f9666))

## [1.1.0](https://www.github.com/little-spoon-dev/design-system/compare/theme-v1.0.0...theme-v1.1.0) (2021-09-16)

### Features

- **theme:** add fonts to theme & storybook LS-1721 ([6f04f27](https://www.github.com/little-spoon-dev/design-system/commit/6f04f272b2006c4c46522af6a186db4b7162475d))

### Bug Fixes

- **theme:** update snapshot, export fonts ([1594f54](https://www.github.com/little-spoon-dev/design-system/commit/1594f5452fa282082ca08a9d2441146951d34f45))

## 1.0.0 (2021-09-15)

### Features

- **theme:** add colors (primary, secondary, alert, token) LS-1710 ([8a195a5](https://www.github.com/little-spoon-dev/design-system/commit/8a195a5243e69baf1e751caf1f634071537e2efd))
- **theme:** create package `@littlespoon/theme` LS-1710 ([72b2efd](https://www.github.com/little-spoon-dev/design-system/commit/72b2efd7f71c5e2533c0c63c15441675a655d6b4))
