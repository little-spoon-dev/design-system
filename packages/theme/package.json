{
  "name": "@littlespoon/theme",
  "version": "1.14.2",
  "description": "Little Spoon theme",
  "main": "lib/index.js",
  "scripts": {
    "build": "tsc",
    "build:flowtypes": "flowgen lib && rm -rf lib && mv export.flow.js lib && yarn build",
    "clean": "rm -rf coverage export.flow.js lib",
    "lint": "eslint .",
    "lint:fix": "yarn lint --fix",
    "lint:tsc": "tsc --noEmit",
    "test": "jest",
    "test:watch": "jest --watch"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/little-spoon-dev/design-system.git",
    "directory": "packages/theme"
  },
  "bugs": {
    "url": "https://github.com/little-spoon-dev/design-system/issues"
  },
  "keywords": [
    "littlespoon",
    "theme",
    "colors",
    "fonts"
  ],
  "files": [
    "lib"
  ],
  "publishConfig": {
    "access": "public"
  },
  "license": "UNLICENSED"
}
