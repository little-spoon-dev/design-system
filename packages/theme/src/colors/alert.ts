import { rgb } from '../utils'

/**
 * Success
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/t/193927}
 */
export const success10 = rgb(230, 241, 238)
export const success20 = rgb(153, 199, 187)
export const success30 = rgb(102, 172, 152)
export const success40 = rgb(51, 144, 118)
export const success50 = rgb(0, 116, 84)

export const success = {
  success10,
  success20,
  success30,
  success40,
  success50,
}

/**
 * Warning
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/t/76323f}
 */
export const warning10 = rgb(254, 243, 233)
export const warning20 = rgb(252, 208, 168)
export const warning30 = rgb(251, 185, 124)
export const warning40 = rgb(249, 161, 81)
export const warning50 = rgb(248, 138, 37)

export const warning = {
  warning10,
  warning20,
  warning30,
  warning40,
  warning50,
}

/**
 * Critical
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/t/07d378}
 */
export const critical10 = rgb(250, 233, 231)
export const critical20 = rgb(237, 169, 159)
export const critical30 = rgb(227, 125, 111)
export const critical40 = rgb(218, 82, 63)
export const critical50 = rgb(209, 39, 15)

export const critical = {
  critical10,
  critical20,
  critical30,
  critical40,
  critical50,
}

/**
 * Informative
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/t/90e5b5}
 */
export const informative10 = rgb(233, 239, 249)
export const informative20 = rgb(169, 193, 231)
export const informative30 = rgb(125, 161, 220)
export const informative40 = rgb(82, 130, 208)
export const informative50 = rgb(39, 99, 196)

export const informative = {
  informative10,
  informative20,
  informative30,
  informative40,
  informative50,
}

export type SuccessColor = keyof typeof success
export type WarningColor = keyof typeof warning
export type CriticalColor = keyof typeof critical
export type InformativeColor = keyof typeof informative
export type AlertColor = SuccessColor | WarningColor | CriticalColor | InformativeColor

export const alertColors = {
  ...success,
  ...warning,
  ...critical,
  ...informative,
}

export default {
  success,
  warning,
  critical,
  informative,
}
