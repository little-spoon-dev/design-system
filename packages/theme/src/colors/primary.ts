import { rgb } from '../utils'

/**
 * Brand Blue
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/t/12d188}
 */
export const brand10 = rgb(230, 252, 250)
export const brand20 = rgb(184, 247, 241)
export const brand30 = rgb(138, 242, 232)
export const brand40 = rgb(92, 237, 223)
export const brand50 = rgb(46, 232, 214)
export const brand60 = rgb(0, 227, 205)
export const brand70 = rgb(0, 186, 168)
export const brand80 = rgb(0, 145, 131)
export const brand90 = rgb(0, 104, 94)
export const brand100 = rgb(0, 64, 57)

export const brand = {
  brand10,
  brand20,
  brand30,
  brand40,
  brand50,
  brand60,
  brand70,
  brand80,
  brand90,
  brand100,
}

/**
 * @deprecated Use brand instead.
 */
export const blue10 = brand10
export const blue20 = brand20
export const blue30 = brand30
export const blue40 = brand40
export const blue50 = brand50
export const blue60 = brand60
export const blue70 = brand70
export const blue80 = brand80
export const blue90 = brand90
export const blue100 = brand100

/**
 * @deprecated Use brand instead.
 */
export const primaryBlue = {
  blue10,
  blue20,
  blue30,
  blue40,
  blue50,
  blue60,
  blue70,
  blue80,
  blue90,
  blue100,
}

export type BrandColor = keyof typeof brand
export type PrimaryBlueColor = keyof typeof primaryBlue
export type PrimaryColor = BrandColor | PrimaryBlueColor

export const primaryColors = {
  ...brand,
  ...primaryBlue,
}

export default {
  brand,
  primaryBlue,
}
