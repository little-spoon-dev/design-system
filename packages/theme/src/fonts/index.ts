export const primary = {
  family: 'Lato, sans-serif',
  variant: {
    p1: '1.25rem',
    p2: '1.125rem',
    p3: '1rem',
    p4: '0.875rem',
    caption1: '0.75rem',
    button1: '1.25rem',
    button2: '1.125rem',
    button3: '1rem',
    button4: '0.875rem',
  },
  weight: {
    normal: 400,
    bold: 700,
  },
}

export const secondary = {
  family: 'Mulish, sans-serif',
  variant: {
    display1: '4.625rem',
    display2: '4.125rem',
    h1: '3.25rem',
    h2: '2.875rem',
    h3: '2.25rem',
    h4: '1.8125rem',
    h5: '1.625rem',
    h6: '1.25rem',
  },
  weight: {
    bold: 700,
    extraBold: 800,
    black: 900,
  },
}

export default { primary, secondary }
