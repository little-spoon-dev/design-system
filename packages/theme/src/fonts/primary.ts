import { fontFamily, rem } from '../utils'
import p from './paragraph'

const buttonLetterSpacing = rem(0.15)

/**
 * Buttons
 *
 * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/t/55397a}
 */
export const button = {
  xlarge: {
    ...p.xlarge,
    letterSpacing: buttonLetterSpacing,
  },
  large: {
    ...p.large,
    letterSpacing: buttonLetterSpacing,
  },
  medium: {
    ...p.medium,
    letterSpacing: buttonLetterSpacing,
  },
  small: {
    ...p.small,
    letterSpacing: buttonLetterSpacing,
  },
} as const

/**
 * Captions
 *
 * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/t/36c339}
 */
export const caption = {
  medium: {
    fontSize: rem(1.2),
    lineHeight: rem(1.8),
  },
} as const

/**
 * Font-family
 *
 * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/t/626dbf}
 */
export const family = fontFamily('Lato')

/**
 * Paragraph
 *
 * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/t/294b11}
 */
export const paragraph = p

/**
 * Font-weight
 *
 * {@link https://zeroheight.com/3ddd0f892/p/211297-typography/t/34024f}
 */
export const weight = {
  normal: 400,
  bold: 700,
} as const

export default {
  button,
  caption,
  family,
  paragraph,
  weight,
}
