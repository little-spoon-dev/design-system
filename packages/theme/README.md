# @littlespoon/theme

[![NPM version](https://img.shields.io/npm/v/@littlespoon/theme.svg)](https://www.npmjs.com/package/@littlespoon/theme)

## Install

[Yarn](https://yarnpkg.com/package/@littlespoon/theme):

```sh
yarn add @littlespoon/theme
```

[npm](https://www.npmjs.com/package/@littlespoon/theme):

```sh
npm install @littlespoon/theme --save
```

[UNPKG](https://unpkg.com/browse/@littlespoon/theme/)

## Usage

Import theme:

```ts
import theme from '@littlespoon/theme'

theme.colors.primary.primaryBlue.blue60() // 'rgba(44,213,196,1)'
```

Import colors:

```ts
import colors from '@littlespoon/theme/lib/colors'

colors.primary.primaryBlue.blue60() // 'rgba(44,213,196,1)'
```

Import color with alpha transparency:

```ts
import { blue60 } from '@littlespoon/theme/lib/colors/primary'

blue60(0.5) // 'rgba(44,213,196,0.5)'
```
