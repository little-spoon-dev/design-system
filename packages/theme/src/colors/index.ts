/**
 * Colors
 *
 * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/b/8982a2}
 */
import alert from './alert'
import primary from './primary'
import secondary from './secondary'
import token from './token'

const colors = {
  ...alert.critical,
  ...alert.informative,
  ...alert.success,
  ...alert.warning,
  ...primary.brand,
  ...primary.primaryBlue,
  ...secondary.secondaryBlue,
  ...secondary.secondaryDeepGreen,
  ...secondary.secondaryGold,
  ...secondary.secondaryLimeGreen,
  ...secondary.secondaryMintGreen,
  ...secondary.secondaryNeutralGrey,
  ...secondary.secondaryPeach,
  ...secondary.secondaryPeach,
  ...secondary.secondaryPink,
  ...token,
  alert,
  primary,
  secondary,
  token,
}

export { alert, primary, secondary, token }
export type Color = keyof typeof colors
export default colors
