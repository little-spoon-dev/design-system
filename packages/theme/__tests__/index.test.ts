import theme, { colors as themeColors, fonts as themeFonts } from '../src/'
import colors from '../src/colors'
import primaryColor from '../src/colors/primary'
import { brand60 } from '../src/colors/primary'
import fonts from '../src/fonts'
import primaryFont from '../src/fonts/primary'
import { family } from '../src/fonts/primary'

it('matches theme', () => {
  expect(theme).toMatchSnapshot()
})

describe('colors', () => {
  it('matches colors', () => {
    expect(theme.colors).toBe(colors)
    expect(themeColors).toBe(colors)
  })

  it('matches primary', () => {
    expect(theme.colors.primary).toBe(primaryColor)
  })

  it('matches brand60', () => {
    expect(theme.colors.primary.brand.brand60).toBe(brand60)
    expect(brand60()).toBe('rgba(0,227,205,1)')
  })
})

describe('fonts', () => {
  it('matches fonts', () => {
    expect(theme.fonts).toBe(fonts)
    expect(themeFonts).toBe(fonts)
  })

  it('matches primary', () => {
    expect(theme.fonts.primary).toBe(primaryFont)
  })

  it('matches family', () => {
    expect(theme.fonts.primary.family).toBe(family)
    expect(family).toBe('Lato, sans-serif')
  })
})
