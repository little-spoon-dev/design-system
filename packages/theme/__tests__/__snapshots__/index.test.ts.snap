// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches theme 1`] = `
Object {
  "colors": Object {
    "alert": Object {
      "critical": Object {
        "critical10": [Function],
        "critical20": [Function],
        "critical30": [Function],
        "critical40": [Function],
        "critical50": [Function],
      },
      "informative": Object {
        "informative10": [Function],
        "informative20": [Function],
        "informative30": [Function],
        "informative40": [Function],
        "informative50": [Function],
      },
      "success": Object {
        "success10": [Function],
        "success20": [Function],
        "success30": [Function],
        "success40": [Function],
        "success50": [Function],
      },
      "warning": Object {
        "warning10": [Function],
        "warning20": [Function],
        "warning30": [Function],
        "warning40": [Function],
        "warning50": [Function],
      },
    },
    "primary": Object {
      "primaryBlue": Object {
        "blue10": [Function],
        "blue100": [Function],
        "blue20": [Function],
        "blue30": [Function],
        "blue40": [Function],
        "blue50": [Function],
        "blue60": [Function],
        "blue70": [Function],
        "blue80": [Function],
        "blue90": [Function],
      },
    },
    "secondary": Object {
      "secondaryDeepGreen": Object {
        "deepGreen10": [Function],
        "deepGreen100": [Function],
        "deepGreen20": [Function],
        "deepGreen30": [Function],
        "deepGreen40": [Function],
        "deepGreen50": [Function],
        "deepGreen60": [Function],
        "deepGreen70": [Function],
        "deepGreen80": [Function],
        "deepGreen90": [Function],
      },
      "secondaryGold": Object {
        "gold10": [Function],
        "gold100": [Function],
        "gold20": [Function],
        "gold30": [Function],
        "gold40": [Function],
        "gold50": [Function],
        "gold60": [Function],
        "gold70": [Function],
        "gold80": [Function],
        "gold90": [Function],
      },
      "secondaryLimeGreen": Object {
        "limeGreen10": [Function],
        "limeGreen100": [Function],
        "limeGreen20": [Function],
        "limeGreen30": [Function],
        "limeGreen40": [Function],
        "limeGreen50": [Function],
        "limeGreen60": [Function],
        "limeGreen70": [Function],
        "limeGreen80": [Function],
        "limeGreen90": [Function],
      },
      "secondaryMintGreen": Object {
        "mintGreen10": [Function],
        "mintGreen100": [Function],
        "mintGreen20": [Function],
        "mintGreen30": [Function],
        "mintGreen40": [Function],
        "mintGreen50": [Function],
        "mintGreen60": [Function],
        "mintGreen70": [Function],
        "mintGreen80": [Function],
        "mintGreen90": [Function],
      },
      "secondaryNeutralGrey": Object {
        "grey10": [Function],
        "grey100": [Function],
        "grey20": [Function],
        "grey30": [Function],
        "grey40": [Function],
        "grey50": [Function],
        "grey60": [Function],
        "grey70": [Function],
        "grey80": [Function],
        "grey90": [Function],
      },
      "secondaryPeach": Object {
        "peach10": [Function],
        "peach100": [Function],
        "peach20": [Function],
        "peach30": [Function],
        "peach40": [Function],
        "peach50": [Function],
        "peach60": [Function],
        "peach70": [Function],
        "peach80": [Function],
        "peach90": [Function],
      },
      "secondaryPink": Object {
        "pink10": [Function],
        "pink100": [Function],
        "pink20": [Function],
        "pink30": [Function],
        "pink40": [Function],
        "pink50": [Function],
        "pink60": [Function],
        "pink70": [Function],
        "pink80": [Function],
        "pink90": [Function],
      },
      "secondaryPurple": Object {
        "purple10": [Function],
        "purple100": [Function],
        "purple20": [Function],
        "purple30": [Function],
        "purple40": [Function],
        "purple50": [Function],
        "purple60": [Function],
        "purple70": [Function],
        "purple80": [Function],
        "purple90": [Function],
      },
      "secondaryRoyalBlue": Object {
        "royalBlue10": [Function],
        "royalBlue100": [Function],
        "royalBlue20": [Function],
        "royalBlue30": [Function],
        "royalBlue40": [Function],
        "royalBlue50": [Function],
        "royalBlue60": [Function],
        "royalBlue70": [Function],
        "royalBlue80": [Function],
        "royalBlue90": [Function],
      },
    },
    "token": Object {
      "borderBold": "rgba(123,123,123,1)",
      "borderContrast": "rgba(66,66,66,1)",
      "borderCritical": "rgba(237,169,159,1)",
      "borderInformative": "rgba(169,193,231,1)",
      "borderMinimal": "rgba(224,224,224,1)",
      "borderModerate": "rgba(173,173,173,1)",
      "borderSubtle": "rgba(205,205,205,1)",
      "borderSuccess": "rgba(153,199,187,1)",
      "borderWarning": "rgba(252,208,168,1)",
      "brand": "rgba(44,213,196,1)",
      "shadeBlack": "#000000",
      "shadeWhite": "#FFFFFF",
      "subSurfaceCritical": "rgba(250,233,231,1)",
      "subSurfaceInformative": "rgba(233,239,249,1)",
      "subSurfaceSuccess": "rgba(230,241,238,1)",
      "subSurfaceWarning": "rgba(254,243,233,1)",
      "surfaceContrast": "rgba(66,66,66,1)",
      "surfaceOverlay": "rgba(66,66,66,0.8)",
      "surfacePrimary": "#FFFFFF",
      "surfaceSecondary": "rgba(241,241,241,1)",
      "surfaceStrong": "#14625A",
    },
  },
  "fonts": Object {
    "primary": Object {
      "family": "Lato, sans-serif",
      "variant": Object {
        "button1": "1.25rem",
        "button2": "1.125rem",
        "button3": "1rem",
        "button4": "0.875rem",
        "caption1": "0.75rem",
        "p1": "1.25rem",
        "p2": "1.125rem",
        "p3": "1rem",
        "p4": "0.875rem",
      },
      "weight": Object {
        "bold": 700,
        "normal": 400,
      },
    },
    "secondary": Object {
      "family": "Mulish, sans-serif",
      "variant": Object {
        "display1": "4.625rem",
        "display2": "4.125rem",
        "h1": "3.25rem",
        "h2": "2.875rem",
        "h3": "2.25rem",
        "h4": "1.8125rem",
        "h5": "1.625rem",
        "h6": "1.25rem",
      },
      "weight": Object {
        "black": 900,
        "bold": 700,
        "extraBold": 800,
      },
    },
  },
}
`;
